lib/tree-view.coffee

```diff
@@ -688,6 +728,8 @@ class TreeView extends View
       fs.moveSync(initialPath, newPath)
 
       if repo = repoForPath(newPath)
+        # The return values of this are thrown away, they are called for emiiter
+        # side effects only.
```

Comment:
`GitRepositoryAsync` has `.refreshStatusForPath` which does this, just with a clearer name.